ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/Main.scala

```diff
@@ -116,33 +118,62 @@ object Main extends StrictLogging {
     logger.info(s"$config")
 
     val et: ET[ExitCode] = for {
-      ledgerId <- either(
-        getLedgerId(config.jwt).leftMap(_ =>
-          new IllegalArgumentException("Cannot infer Ledger ID from JWT")
-        )
-      ): ET[LedgerId]
 
       _ <- either(
         config.traverse(s => resolveSimulationClass(s))
       ): ET[Config[Class[_ <: Simulation]]]
 
       exitCode <- rightT(
-        main2(ledgerId, config)
+        main2(config)
       ): ET[ExitCode]
 
     } yield exitCode
 
     et.run
   }
 
-  private def main2(ledgerId: LedgerId, config: Config[String])(implicit
+  private def main2(config: Config[String])(implicit
```

Comment:
main1 and main2 are a bit strange names (for a newcomer like me) can this be more descriptive?

Reply:
I agree... Had the exact same thought already too